contrib_bots/lib/github_detail.md

```diff
@@ -0,0 +1,28 @@
+# GitHub detail bot
+This bot links and details issues and pull requests.
+To use it username mention the bot then type an id:
+Ids can be specified in three different forms:
+- Id only: `#2000`
+- Repository and id: `zulip#2000`
+- Owner, repository and id `zulip/zulip#2000`
+
+Both the username mention and the id can occur at any time in the message. You
+can also mention multiple ids in a single message.
+
+The bot *requires* a default owner and repository to be configured.
+The configuration file should be located at `~/.contrib_bots/github_detail.ini`.
+It should look like this:
+```ini
+[GitHub]
```

Comment:
@trueskawka Ah, I did this accidentally because of the linter which says it's a misspelling. How do I get around this?